cms/static/js/views/paging_header.js

```diff
@@ -78,11 +83,11 @@ define(["underscore", "backbone", "gettext", "text!templates/paging-header.under
             totalItemsCountLabel: function() {
                 var totalItemsLabel;
                 // Translators: turns into "25 total" to be used in other sentences, e.g. "Showing 0-9 out of 25 total".
-                totalItemsLabel = interpolate(gettext('%(total_items)s total'), {
-                    total_items: this.view.collection.totalCount
+                totalItemsLabel = interpolate(gettext('%(totalItems)s total'), {
```

Comment:
Since you are changing these, can you convert this code to use the safe template best practices, i.e. http://ui-toolkit.edx.org/utilities/string-utils/